### PR TITLE
Fixes #10987: Restrict to Foreman 1.9

### PR DESCRIPTION
**Pull request by ehelms**
There are some functionality we are now using that are not available
in Foreman 1.8 and below such as the logging implementation.

### CODE REVIEW

**Comment by jlsherrill** [comment]
APJ

**Comment by ehelms** [comment]
[test]

File: lib/katello/plugin.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Foreman::Plugin.register :katello do
-  requires_foreman '> 1.3'
+  requires_foreman '> 1.9'
```
Comment (elyezer) [inline lib/katello/plugin.rb]: This should not be >=?

Comment (ehelms) [inline lib/katello/plugin.rb]: Doh .. good catch.

Comment (ehelms) [inline lib/katello/plugin.rb]: Mind filing an issue on that and I'l lfix it? Or you can open PR.

**Pull request merged**